src/crate/client/connection.py

```diff
@@ -146,6 +148,7 @@ def connect(servers=None,
     """
     return Connection(servers=servers,
                       timeout=timeout,
+                      backoff_factor=backoff_factor,
```

Comment:
The comments above document the parameters for the Connection. Please document the `backoff_factor` in the same way as above.